src/FirstSteps/RPG/Adventures/Hell.cs

```diff
@@ -0,0 +1,68 @@
+﻿using FirstSteps.RPG.Heroes;
+using FirstSteps.RPG.Tools;
+using Spectre.Console;
+using System;
+
+namespace FirstSteps.RPG.Adventures
+{
+    public class Hell
```

Comment:
![image](https://user-images.githubusercontent.com/10303508/227742059-5dd17555-e1ff-4114-8b16-c84c08d200c9.png)
Point 7. from the issue: `If the player loses all health, the game should display information about their death and stop.` Now I can play after death xD